rs/protobuf/def/registry/subnet/v1/subnet.proto

```diff
@@ -98,7 +98,10 @@ message EcdsaInitialization {
 
 message ChainKeyInitialization {
   registry.crypto.v1.MasterPublicKeyId key_id = 1;
-  InitialIDkgDealings dealings = 2;
+  oneof initialization {
+    InitialIDkgDealings dealings = 2;
+    InitialNiDkgTranscriptRecord transcript = 3;
```

Comment:
Noticing that the naming is not consistent, please double check that this is what you want.

For consistency, it should be

```suggestion
    InitialNiDkgTranscriptRecord transcript_record = 3;
```

or maybe 

```suggestion
    InitialNiDkgTranscript transcript = 3;
```